modules/op-creating-pipeline-run-pac.adoc

```diff
@@ -0,0 +1,157 @@
++
+.Example of matching a pipeline run to a pull request event
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-pr-main
+annotations:
+  pipelinesascode.tekton.dev/on-target-branch: "[main]"
+  pipelinesascode.tekton.dev/on-event: "[pull_request]"
+# ...
+----
+
+** A `pipelinesascode.tekton.dev/on-comment` annotation, which matches the pipeline run to a comment on a pull request by regular expression. If you match your pipeline run to a comment, it starts when the comment is added to a pull request. To start the pipeline run again. add the comment again.
++
+.Example of matching a pipeline run to a comment
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-comment
+annotations:
+  pipelinesascode.tekton.dev/on-comment: "^/merge-pr"
+# ...
+----
+
+. Optional: Add one or more annotations that filter the matching events. With these annotations, when the defined matching event (such as a pull request, a push event. or a comment) happens, {pac} checks if these annotations are also matched. The pipeline run starts only if all the annotations that you added to it are matched. See the "Annotations for filtering events" section for detailed information about these annotations.
+
+** A `pipelinesascode.tekton.dev/on-path-changed` annotation is matched if the pull request or push event affect files in the specified paths.
+
+** A `pipelinesascode.tekton.dev/on-path-changed-ignore` annotation excludes matching the event if it affects _only_ files in the specified paths.
+
+** A `pipelinesascode.tekton.dev/on-label` annotation is matched if the pull request or push event has one of the specified labels.
+
+** A `pipelinesascode.tekton.dev/on-cel-expression` annotation is matched if the pull request or push event matches a Common Expression Language (CEL) expression.
++
+.Example of using annotations to filter matching events in a pipeline run
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-pr-filtered
+annotations:
+  pipelinesascode.tekton.dev/on-target-branch: "[main]"
+  pipelinesascode.tekton.dev/on-event: "[pull_request]"
+  pipelinesascode.tekton.dev/on-label: "[bug, defect]"
+  pipelinesascode.tekton.dev/on-path-changed: "["src/*", "cli/**"]"
+# ...
+----
+
+. Optional: Add the `pipelinesascode.tekton.dev/cancel-in-progress: "true"` annotation to enable automatic cancellation of the pipeline run in certain cases. For example, if a pull request triggers a pipeline run and then the user pushes new commits into the pull request source branch, each push triggers a new copy of the pipeline run. if you enable automatic cancellation-in-progress, after a new copy of the pipeline run starts, {pac} cancels the older run ti avoid running many copies of the pipeline run at the same time. See the "Annotations for specifying automatic cancellation-in-progress for a pipeline run" for detailed information about this annotation.
```

Comment:
```suggestion
. Optional: Add the `pipelinesascode.tekton.dev/cancel-in-progress: "true"` annotation to enable automatic cancellation of the pipeline run in certain cases. For example, if a pull request triggers a pipeline run and then the user pushes new commits into the pull request source branch, each push triggers a new copy of the pipeline run. if you enable automatic cancellation-in-progress, after a new copy of the pipeline run starts, {pac} cancels the older run to avoid running many copies of the pipeline run at the same time. See the "Annotations for specifying automatic cancellation-in-progress for a pipeline run" for detailed information about this annotation.
```